src/main/service/listParseHelperService.ts

```diff
@@ -0,0 +1,359 @@
+  public findAndManipulateJudiciary(session: object): string {
+    let judiciaries = '';
+    let foundPresiding = false;
+    session['judiciary']?.forEach(judiciary => {
+      if (judiciary?.isPresiding === true) {
+        judiciaries = ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs);
+        foundPresiding = true;
+      } else if (!foundPresiding) {
+        if (ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs) !== '') {
+          judiciaries += ListParseHelperService.writeStringIfValid(judiciary?.johKnownAs) + ', ';
+        }
+      }
+    });
+
+    if (!foundPresiding) {
+      judiciaries = judiciaries.slice(0, -2);
+    }
+
+    return judiciaries;
+  }
+
+  /**
+   * Format linked cases by joining individual case ID with delimiter
+   * @param hearing
+   */
+  public findAndManipulateLinkedCases(hearing: object): void {
+    hearing['case'].forEach(hearingCase => {
+      let linkedCases = '';
+      let counter = 1;
+      hearingCase['caseLinked']?.forEach(linkedCase => {
+        linkedCases += (counter == hearingCase['caseLinked'].length)
+          ? linkedCase['caseId']
+          : linkedCase['caseId'] + ', ';
+        counter++;
+      });
+      hearingCase['formattedLinkedCases'] = linkedCases;
+    });
+  }
+
+  /**
+   * Calculate the duration of a sitting.
+   * @param sitting
+   */
+  public calculateDuration(sitting: object): void {
+    sitting['duration'] = '';
+    sitting['startTime'] = '';
+    if (sitting['sittingStart'] !== '' && sitting['sittingEnd'] !== '') {
+      const sittingStart = moment.utc(sitting['sittingStart']);
+      const sittingEnd = moment.utc(sitting['sittingEnd']);
+      let durationAsHours = 0;
+      let durationAsMinutes = moment.duration(sittingEnd.startOf('minutes').diff(sittingStart.startOf('minutes'))).asMinutes();
+      if (durationAsMinutes >= 60) {
+        durationAsHours = Math.floor(durationAsMinutes / 60);
+        durationAsMinutes = durationAsMinutes - (durationAsHours * 60);
+      }
+
+      let durationAsDays = 0;
+      if(durationAsHours >= 24) {
+        durationAsDays = Math.floor(durationAsHours / 24);
+      }
+
+      sitting['durationAsHours'] = durationAsHours;
+      sitting['durationAsMinutes'] = durationAsMinutes;
+      sitting['durationAsDays'] = durationAsDays;
+
+      sitting['time'] = moment.utc(sitting['sittingStart']).tz(this.timeZone).format('HH:mm');
+      const min = moment(sitting['sittingStart'], 'HH:mm').minutes();
+      if (min === 0) {
+        sitting['startTime'] = moment.utc(sitting['sittingStart']).tz(this.timeZone).format('ha');
+      } else {
+        sitting['startTime'] = moment.utc(sitting['sittingStart']).tz(this.timeZone).format('h.mma');
+      }
+    }
+  }
+
+  /**
+   * Function which extracts the time from a UTC Date Time in BST format.
+   * @param publicationDatetime The publication date time to convert in UTC.
+   */
+  public publicationTimeInBst(publicationDatetime: string): string {
```

Comment:
The terminology used for this method and publicationDateInBst is not correctly. Wonder we should update them? What these methods doing are update the time to UK Local time and not just BST. Currently we are not in daylight saving but in GMT time and this method is returning datetime in GMT now. In the summer it is right that we are retuning BST time. May be can rename it to publicationTimeInUKLocalTime or something else more appropriate?